lib/src/builder.dart

```diff
@@ -75,9 +75,9 @@ class AggregateTestBuilder extends Builder {
   @override
   Future<void> build(BuildStep buildStep) async {
     _config ??= await decodeConfig(buildStep);
-    if (!_config.browserAggregation) {
+    if (!_config!.browserAggregation) {
```

Comment:
I feel like you ought to be able to use a single ! somewhere here to assert the config's not nil in one place. I think just `(await decodeConfig(buildStep))! `will work.

Reply:
This didn't work unfortunately because `decodeConfig`'s return type is non-nullable, so the analyzer complains about it being unnecessary. But I decided to use a local non-nullable var to avoid having to use `!` everywhere below.